<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rekall</class>
 <widget class="QMainWindow" name="Rekall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rekall</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget, QLabel, QCheckBox, QLineEdit, QPlainTextEdit, QPushButton, QSpinBox, QDoubleSpinBox, QTreeView, QHeaderView, QTabBar, QComboBox, QFrame#globalFrame, QTabBar::tab, QTreeView, QHeaderView, QDockWidget, QStatusBar, QRadioButton, QToolBox, QToolBox::tab {
	font: 						10px &quot;Calibri&quot;, &quot;Arial&quot;;
	padding:					0px;
	margin:					0px;
	min-height: 				20px;
	color: 						rgb(245, 248, 250);
	border: 					0px solid black;
}
QPushButton, QSpinBox, QDoubleSpinBox, QLineEdit, QPlainTextEdit, QCheckBox::indicator, QRadioButton::indicator, QTreeView::indicator, QComboBox, QTreeView, QHeaderView {
	border: 					1px solid rgb(55, 55, 55);
	border-radius: 		2px;
	background-color: 	rgb(71, 77, 79);
}
QSpinBox, QDoubleSpinBox, QLineEdit, QPlainTextEdit, QCheckBox {
	margin-top: 			4px;
	margin-bottom: 		4px;
}

QDialog, QWidget#chutier, QFrame#globalFrame, QWidget#centralwidget, QWidget#player, QWidget#timeline, QWidget#viewer {
	background-color: 	rgb(56, 61, 63);
	border:						0px solid black;
}

QFrame#globalFrameBlk {
	background-color: 	rgb(0, 0, 0);
	border:						0px solid black;
}


QLabel, QCheckBox, QRadioButton {
	color: 						rgb(245, 248, 250);
}



QTabWidget::pane {
	border: 					1px solid rgb(50, 50, 50);
}
QTabWidget::tab-bar {
	left: 							0px;
}
QTabWidget#tab {
	color: red;
}
QTabBar::tab {
	min-height: 							25px; /* FIX IN SS-TAB */
	background-color: 				rgb(110, 110, 110);
	color: 									rgb(220, 220, 220);
    border-top-left-radius: 		4px;
    border-top-right-radius: 	4px;
	margin-right:						2px;
	padding-left: 						5px;
	padding-right:						5px;
}
QTabBar::tab:selected {
	background-color: 				rgb(242, 241, 237);
	color: 									black;
}
QTabBar::tab:hover {
	color: 									black;
}
QTabBar::tab:!selected {
    margin-top: 						2px;
}
QTabBar::tab:disabled {
	color: 									gray;
}


QToolBox::tab {
	background-color: 				rgb(41, 44, 45);
	color: 									rgb(194, 198, 198);
}
QToolBox::tab:selected {
	background-color: 				rgb(56, 61, 63);
}
QToolBox::tab:disabled {
	background-color: 				rgb(41, 44, 45);
	color: 									rgb(56, 61, 63);
}
QToolBox QWidget#chutierPage, QToolBox QWidget#feedPage, QToolBox QWidget#infosPage, QToolBox QWidget#conduitePage, QToolBox QWidget#tasksPage, QToolBox QWidget#peoplePage, QToolBox QWidget#previewPage {
	background-color: 				rgb(56, 61, 63);
}


QTreeView {
	background:								rgb(56, 61, 63);
	alternate-background-color: 	rgb(56, 61, 63);
	selection-background-color: 	rgb(41, 44, 45);
	gridline-color: 							rgb(56, 61, 63);
	border:										0px solid black;
}
QHeaderView {
	background: 							transparent;
	border: 									0px solid black;
	color: 										rgb(200, 200, 200);
	min-height: 								14px;
}
QHeaderView::section {
	background: 							rgb(56, 61, 63);
	border: 									0px solid transparent;
	border-bottom: 						1px solid rgb(41, 44, 45);
	font: 										9px;
}
QTreeView::item {
	min-height: 		20px;
}
QTreeView::branch:has-children:!has-siblings:closed, QTreeView::branch:closed:has-children:has-siblings {
	border-image: 	none;
	image: 					url(:/items/res_tree_close.png);
}
QTreeView::branch:open:has-children:!has-siblings, QTreeView::branch:open:has-children:has-siblings  {
	border-image: 	none;
	image: 					url(:/items/res_tree_open.png);
}




QPushButton {
	margin-left: 			6px;
}
QPushButton:hover {
	border: 					1px solid rgb(28, 124, 195);
	background-color: 	rgb(21, 91, 143);
}
QPushButton::checked, QPushButton:pressed, QCheckBox::indicator:checked, QTreeView::indicator:checked, QRadioButton::indicator:checked {
	background-color: 	rgb(0, 187, 255);
}


QCheckBox, QRadioButton {
	spacing: 					5px;
}
QCheckBox::indicator, QTreeView::indicator, QRadioButton::indicator  {
	width: 						10px;
	height: 					10px;
}


QSpinBox, QDoubleSpinBox, QLineEdit {
	padding-left	:			5px;
}
QSpinBox::up-button, QDoubleSpinBox::up-button, QSpinBox::down-button, QDoubleSpinBox::down-button {
	width: 						5px;
	padding: 					2px;
	padding-left:			3px;
	background-color: 	rgb(60, 60, 60);
	border: 					0px solid black;
	border-left:				1px solid rgb(70, 70, 70);
}
QSpinBox::up-button, QDoubleSpinBox::up-button {
    image: 						url(:/icons/res_icon_plus.png);
	border-bottom:		1px solid rgb(70, 70, 70);
}
QSpinBox::down-button, QDoubleSpinBox::down-button {
    image: 						url(:/icons/res_icon_minus.png);
}


QComboBox {
	background:			rgb(242, 241, 237);
	color: 					black;
	margin-left: 		3px;
	border-radius: 	3px;
	selection-color: 	black;
	padding-left: 		5px;
}
QComboBox::drop-down {
	width: 						10px;
	padding: 					2px;
	border: 					0px solid black;
	background-color: 	rgb(70, 70, 70);
    image: 						url(:/icons/res_icon_down.png);
}



QScrollBar:vertical, QScrollBar:horizontal {
	border: 				0px solid black;
	background: 		rgba(255, 255, 255, 32);
}
QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
	border: 				0px solid black;
	background: 		rgb(161, 161, 161);
	border-radius: 	3px;
}
QScrollBar:vertical {						width: 			8px;					}
QScrollBar:horizontal {					height: 		8px;				}
QScrollBar::handle:vertical {			min-height: 8px;		}
QScrollBar::handle:horizontal {	min-width: 	8px;			}
QScrollBar::add-line:vertical, QScrollBar::add-line:hozirontal, QScrollBar::sub-line:vertical, QScrollBar::sub-line:horizontal, QScrollBar::up-arrow:vertical, QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:vertical, QScrollBar::down-arrow:horizontal {
	border: 			0px solid black;
	background: 	transparent;
	width: 				1px;
	height: 			1px;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical, QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: 	none;
}


QSlider:horizontal {
	min-width: 				100px;
}
QSlider::groove:horizontal {
	border: 					0px solid transparent;
	background-color: 	rgb(70, 70, 70);
	height: 					3px;
}
QSlider::handle:horizontal {
	width: 						11px;
	height: 					10px;
	border-radius: 		5px;
	margin-top: 			-4px;
	margin-bottom: 		-4px;
	border: 					0px solid black;
	background-color: 	rgb(0, 187, 250);
}


QSplitter::handle {
	background: 	rgb(56, 61, 63);
}
QSplitter::handle:pressed {
	background: 	rgb(50, 221, 255);
 }
QSplitter::handle:horizontal	 	{	width: 	4px; }
QSplitter::handle:vertical     		{	height:	4px; }

QDockWidget {
}
QDockWidget::title {
    text-align:				center;
	background-color: 	rgb(70, 70, 70);
}


QToolButton {
	border: 					0px solid black;
	border-radius: 		3px;
	margin: 					2px;
}
QToolButton:pressed, QToolButton:checked {
	border: 					0px solid black;
    background-color: 	rgba(0, 0, 0, 20);
}


QStatusBar {
	color: 						rgb(210, 210, 210);
	border-top: 				1px solid rgb(50, 50, 50);
	background-color: 	rgb(30, 30, 30);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="conduiteSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="timelineSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="fileSplitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QToolBox" name="toolBoxLeft">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="tabSpacing">
          <number>0</number>
         </property>
         <widget class="QWidget" name="chutierPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>136</width>
            <height>260</height>
           </rect>
          </property>
          <attribute name="label">
           <string>FILES</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="UiTreeView" name="chutier" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="feedPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>136</width>
            <height>260</height>
           </rect>
          </property>
          <attribute name="label">
           <string>EVENTS</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="FeedList" name="feeds" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tasksPage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>136</width>
            <height>260</height>
           </rect>
          </property>
          <attribute name="label">
           <string>PROCESSINGS</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="TasksList" name="tasks" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="peoplePage">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>136</width>
            <height>260</height>
           </rect>
          </property>
          <attribute name="label">
           <string>PEOPLE</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
           <item>
            <widget class="QTreeWidget" name="persons">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="indentation">
              <number>0</number>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Photo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="Player" name="player" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </widget>
       <widget class="Timeline" name="timeline" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QToolBox" name="toolBoxRight">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabSpacing">
        <number>5</number>
       </property>
       <widget class="QWidget" name="infosPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>770</width>
          <height>483</height>
         </rect>
        </property>
        <attribute name="label">
         <string>INFOS</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="Previewer" name="preview" native="true"/>
         </item>
         <item>
          <widget class="QTreeWidget" name="metadata">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="indentation">
            <number>10</number>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">Key</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="metadataOpenGps">
           <property name="text">
            <string>SHOW GPS LOCATION ON A MAP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="metadataSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="conduitePage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>100</height>
         </rect>
        </property>
        <attribute name="label">
         <string>CONDUITE</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="Viewer" name="viewer" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionRewind"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionInspector"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuWindow"/>
  </widget>
  <action name="actionPlay">
   <property name="text">
    <string>Play/pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionRewind">
   <property name="text">
    <string>Rewind</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionInspector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Inspector</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UiTreeView</class>
   <extends>QWidget</extends>
   <header>items/uitreeview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Timeline</class>
   <extends>QWidget</extends>
   <header>gui/timeline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Player</class>
   <extends>QWidget</extends>
   <header>gui/player.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>gui/viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FeedList</class>
   <extends>QWidget</extends>
   <header>tasks/feedlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TasksList</class>
   <extends>QWidget</extends>
   <header>tasks/taskslist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Previewer</class>
   <extends>QWidget</extends>
   <header>gui/previewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>metadata</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>Rekall</receiver>
   <slot>actionMetadata(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1024</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>916</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metadataSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Rekall</receiver>
   <slot>refreshMetadata()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1024</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>-62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>metadataOpenGps</sender>
   <signal>released()</signal>
   <receiver>Rekall</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>1145</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>persons</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>Rekall</receiver>
   <slot>personItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>-71</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action()</slot>
  <slot>actionMetadata(QTreeWidgetItem*,int)</slot>
  <slot>refreshMetadata()</slot>
  <slot>personItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
 </slots>
</ui>
